src/ontology/uberon.Makefile

```diff
@@ -1998,7 +1473,6 @@ normalise_release_serialisation_ofn:
 
 normalise_release_serialisation_rdfmxml:
```

Comment:
`normalise_release`, `normalise_release_serialisation_rdfmxml` and `normalise_release_serialisation_ofn` are remnants of an older OWLAPI serialisation and have not been needed since ODK 1.4.0.

Given these release instructions: http://obophenotype.github.io/uberon/uberon-release/

the file called /Users/matentzn/ws/uberon/src/ontology/run-release.sh is obsolete and should also be deleted.

Reply:
I’ve left all those in the PURGATORY section at the end (which is basically where I left all the cruft that I didn’t know what to do with). Happy to make another pass in that section to drop those.